pandas/core/arrays/datetimes.py

```diff
@@ -264,7 +264,10 @@ def _validate_dtype(cls, values, dtype):
     # error: Signature of "_simple_new" incompatible with supertype "NDArrayBacked"
     @classmethod
     def _simple_new(  # type: ignore[override]
-        cls, values: np.ndarray, freq: BaseOffset | None = None, dtype=DT64NS_DTYPE
+        cls,
+        values: np.ndarray,
+        freq: str | BaseOffset | lib.NoDefault | None = None,
```

Comment:
@jbrockmendel should the caller ensure the `freq` is not a string when calling `_simple_new`?

Reply:
this should be `BaseOffset | None`

Reply:
Thank you, I changed it!